SmartDeviceLink/SDLSetInteriorVehicleDataResponse.h

```diff
@@ -0,0 +1,20 @@
+//
+//  SDLSetInteriorVehicleDataResponse.h
+//
+
+#import "SDLRPCResponse.h"
+@class SDLModuleData;
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * Used to set the values of one remote control module
+ *
+ */
+@interface SDLSetInteriorVehicleDataResponse : SDLRPCResponse
+
+@property (strong, nonatomic) SDLModuleData *moduleData;
```

Comment:
What does this mean, is this the new module data after the attempted set? Needs documentation